docs/distribution/distribution.md

```diff
@@ -12,52 +12,130 @@ be structured and how it should behave.
 
 <!-- MarkdownTOC levels="2,3" autolink="true" -->
 
-- [Enso Home Layout](#enso-home-layout)
 - [Universal Launcher Script](#universal-launcher-script)
+- [Enso Distribution Layout](#enso-distribution-layout)
+  - [Portable Enso Distribution Layout](#portable-enso-distribution-layout)
+  - [Installed Enso Distribution Layout](#installed-enso-distribution-layout)
+  - [Converting a Portable Distribution into Installed](#converting-a-portable-distribution-into-installed)
 - [Layout of an Enso Version Package](#layout-of-an-enso-version-package)
   - [Standard Library](#standard-library)
   - [Resolvers](#resolvers)
 
 <!-- /MarkdownTOC -->
 
-## Enso Home Layout
-All of Enso's binaries, packages, etc. are installed into a directory in
-the user's home directory. For macOS and linux distributions that's `~/.enso`,
-by default. The distribution is fully portable, so it never makes any
-assumptions about actually being placed in any particular directory.
+## Universal Launcher Script
```

Comment:
I'm not sure we need to call this a 'script' any more. The "Universal Launcher" is fine!